extras/gce-ingress/frontend.yaml

```diff
@@ -0,0 +1,33 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START gke_boa_kubernetes_manifests_frontend_ingress_frontend]
+apiVersion: networking.k8s.io/v1
+kind: Ingress
```

Comment:
Was there something that wasn't working with the current frontend LB that requires an Ingress? (https://github.com/GoogleCloudPlatform/bank-of-anthos/blob/main/kubernetes-manifests/frontend.yaml#L139-L150)

If so, what are those reasons?

Reply:
The reason is that we use the external metric `loadbalancing.googleapis.com|https|request_count` which came from LB type HTTP(S). With a Service object type LoadBalancer, it creates Network (TCP and UDP). By using the Ingress object GCE ingress controller creates exactly what we need.